OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectWorld.cs

```diff
@@ -19,6 +22,27 @@ sealed class DreamMetaObjectWorld : IDreamMetaObject {
         private ViewRange _viewRange;
 
         private double TickLag => _gameTiming.TickPeriod.TotalMilliseconds / 100;
+        /// <summary> Determines whether we try to show IPv6 or IPv4 to the user during .address and .internet_address queries.</summary>
+        private bool Display_IPv6
```

Comment:
This doesn't fit the naming convention for properties and should be renamed to `DisplayIPv6`

Reply:
I seem to be incapable of restraining the snake-case demon within my heart